### DISCUSSION TITLE
how can I format json to import with LUUID types

### DISCUSSION THREAD

**Discussion by maga-polito** [pkosiec/mongo-seeding#166] [Q&A]
I need to import objects like:
{
    "_id" : LUUID("d8b30e31-5af3-2a4a-bd1c-e478e63940d5"),
    "CreationDate" : ISODate("2019-06-20T15:00:04.913Z")
}

but when looking at documentation I was not able to parse LUUID to the db, any idea?

**Comment by pkosiec** [comment]
HI @maga-polito,

Do you use `js` or `ts` files to import such data? From JSON I think it couldn't be done.

If the code you provided is from JS file, what is your import for `LUUID`?.

Alternatively, you can also try some instructions here: https://stackoverflow.com/questions/25789342/converting-mongodb-binary-id-to-luuid-using-node

Cheers!
